model/gen-ai/registry.yaml

```diff
@@ -226,6 +226,11 @@ groups:
         type: string
         brief: Free-form description of the GenAI agent provided by the application.
         examples: ["Helps with math problems", "Generates fiction stories"]
+      - id: gen_ai.agent.session_id
+        stability: development
+        type: string
+        brief: The unique identifier of the session. Use the same value across requests to continue the same conversation..
+        examples: ["ssn_5j66UpCpwteGg4YSxUnt7lPY"]
```

Comment:
nit, I get a heart attack any time I see `ssn` near telemetry, can we pick another example? 😅

Reply:
Done